mk/external.mk

```diff
@@ -21,21 +21,32 @@ define epilogue
 endef
 
 # $(1): compressed source URL
+# if URL contains git command, then use git
 define download
-    $(eval _ := $(shell wget -q --show-progress --continue "$(strip $(1))"))
+    $(eval _ :=  \
+	$(if $(findstring git,$(1)), \
```

Comment:
I am not sure if this check is correct. What if loading from gist/github without `git` commands?

Reply:
> I am not sure if this check is correct. What if downloading from gist/github without `git` commands?

We might support multiple clients, but currently, we only support the one in use (`wget` and `git clone`). Therefore, we may not encounter non-supported client.